packages/react/.npmrc

```diff
@@ -0,0 +1 @@
+scripts-prepend-node-path=true
```

Comment:
Could you explain how this fixes the build?

Reply:
I basically followed the deployment logs 😄 
https://zeit.co/heetch-eng/flamingo/qz1yd047g